MobileWallet/TariLib/Wrappers/Transactions/CompletedTx.swift

```diff
@@ -136,6 +136,18 @@ class CompletedTx: TxProtocol {
         return (PublicKey(pointer: resultPointer!), nil)
     }
 
+    var transactionKernel: (CompletedTxKernel?, Error?) {
+        var errorCode: Int32 = -1
+        let resultPointer = withUnsafeMutablePointer(to: &errorCode, { error in
+            completed_transaction_get_transaction_kernel(ptr, error)
+        })
```

Comment:
Please use trailing closure.

Reply:
This handling is the same throughout the file?